README.md

```diff
@@ -1,42 +1,60 @@
 # Unreal Engine plugin for Visual Studio
 
-This project contains an Unreal Editor plugin that works in conjunction with Visual Studio to display information about Blueprints assets in C++ code. 
+This project contains an Unreal Editor plugin that works in conjunction with Visual Studio to help discover and run tests in C++ code. 
 
 The plugin can be installed in either the Engine or Game project sources, and it is automatically activated when an Unreal Engine project is opened in Visual Studio.
 
 ## Requirements
 
 Before you begin, please make sure you have the following software and tools set up:
 
-1. Visual Studio 2022 has the "IDE features for Unreal Engine" component installed.
+1. Visual Studio 2022 has the "Visual Studio Tools for Unreal Engine" component installed.
    1. The component can be found in the "Game development with C++" workload or as an individual component.
 2. Unreal Engine, either installed or built from source.
    1. To learn how to install or build Unreal Engine, please refer to the following guide: [Installing Unreal Engine](https://docs.unrealengine.com/5.0/en-US/installing-unreal-engine).
-   1. The source code and instructions have been tested on Unreal Engine versions 4.27, 5.0 and 5.1.
+   1. The source code and instructions have been tested on Unreal Engine versions 4.27 and 5.0+.
 
 ## Building and Installing the Plugin
 
 > If you have Unreal Engine installed and set up through the Epic Games Launcher, and you only want to use the plugin, you can skip the steps below and install it directly from the [Unreal Engine Marketplace](https://aka.ms/vsueplugin).
 
 The most straightforward way to use the plugin is to clone the repo under the `Plugins` folder of your game project or engine source. If you have multiple projects in the same Visual Studio solution, it is recommended to install the plugin at the engine level and share the binaries across the projects.
 
-1. Clone the repo under the project plugin folder by using the following commands:
-
+1. Clone the repo by using the following commands:
    ```powershell
-   cd <Project or Engine root folder>/Plugins
    git clone https://github.com/microsoft/vc-ue-extensions.git
    ```
 
+2. Build the plugin from source:
```

Comment:
We should document the approach we plan to use during the auto-install (from the config page) to unify our approach across all platforms.
First confirm that it works (ideally for all supported UE versions).

Reply:
this section is correct - if anyone wants to build plugin they should run commands specified 
alternately they can copy it to project plugin folder (only project, not engine) and let engine build it

I deliberately not mention it not to complicate docs but can specify if you think it is important

as for config page it wil:
- copy to plugins folder and let engine build in case target is a project;
- build and then copy to plugin folder if target is engine;

Reply:
Makes sense, thanks